include/gatekeeper_cps.h

```diff
@@ -39,6 +39,18 @@ struct cps_config {
 	 */
 	int               debug;
 
+	/* Number of times to attempt bring a KNI interface up or down. */
+	uint8_t           num_attempts_kni_link_set;
```

Comment:
I don't think there's any reason to limit this to a `uint8_t` -- I think `unsigned int` is fine.